src/Player/PlayerBindKeeper.h

```diff
@@ -0,0 +1,14 @@
+#pragma once
+
+namespace al {
+class HitSensor;
+}
+
+class PlayerBindKeeper {
+public:
+    al::HitSensor* getSecondHitSensor() const { return mSecondHitSensor; }
+
+private:
+    char padding[8];
+    al::HitSensor* mSecondHitSensor;
```

Comment:
I’ve been investigating `PlayerBindKeeper` in a separate branch.

It has 2 hit sensors: the first belongs to the player and never changes; and the second changes during certain actions (examples: sitting on a bench, riding a jaxi).

Hence calling this field **_second_** hit sensor.